test/libsolidity/smtCheckerTests/literalSuffixes/equivalent_function_call_with_two_arguments.sol

```diff
@@ -0,0 +1,16 @@
+function suffix(uint m, uint e) pure suffix returns (uint) {
+    return m + e;
+}
+
+contract C {
+    function test() public pure {
+        assert(1.23 suffix == suffix(123, 2));
+        assert(123 suffix == suffix(123, 0));
+        assert(0 suffix == suffix(0, 0));
+    }
```

Comment:
Can you also add some assertions that should fail?

Reply:
I added them in a separate file `equivalent_function_call_with_two_arguments_fail.sol`, ok?